google-cloud-video_intelligence/google-cloud-video_intelligence.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |gem|
 
   gem.add_development_dependency "minitest", "~> 5.10"
   gem.add_development_dependency "redcarpet", "~> 3.0"
-  gem.add_development_dependency "rubocop", "~> 0.59.2"
+  gem.add_development_dependency "rubocop", "~> 0.50.0"
```

Comment:
@dazuma do we need to add rubocop to the synth.py file?

Reply:
Ah, crap, yes we do. I forgot about this. Graham already added it to tasks, I believe. We need to do the same for the other gapic-only libraries that are generating the gemspec.